imutils/video/videostream.py

```diff
@@ -3,7 +3,7 @@
 
 class VideoStream:
 	def __init__(self, src=0, usePiCamera=False, resolution=(320, 240),
-		framerate=32, **kwargs):
+		framerate=32, **kwargs, cap=None):
```

Comment:
This line causes a `SyntaxError`, because you have a variable after capturing all kwargs. I'm not sure why you want to have this variable at all though.

Reply:
Ahh, I wasn't aware of this (relatively new python dev here). Thanks for teaching me.

I had thought adding it at the end of the parameter list would help maintain backwards compatibility.  But since this doesn't work, it looks like we have to overload the `src` arg as an `int or VideoCapture` then and do the stream initialization and property setting only if it was an `int`. This is per the maintainer's suggestion in [#68 (comment)](https://github.com/jrosebr1/imutils/pull/68#issuecomment-543759112)